dependencies/vaticle/repositories.bzl

```diff
@@ -21,5 +21,5 @@ def vaticle_dependencies():
     git_repository(
```

Comment:
Update to dependencies that gives us access to the token-based `deploy_npm` rule